src/components/Label/Label.spec.tsx

```diff
@@ -35,4 +43,12 @@ describe('Label', () => {
             background-color: ${Colors.NEGATIVE_ORANGE_50};
         `);
     });
+
+    it('uses the filled version if set', () => {
+        expect(render(<Label variant="success" filled />).container.firstChild).toHaveStyle(`
+            color: ${Colors.WHITE};
+            border-color: ${Colors.POSITIVE_GREEN_900};
+            backgroundColor: ${Colors.POSITIVE_GREEN_900};
```

Comment:
This should be `background-color`.

Reply:
Sorry that it's in a different PR :( https://github.com/freenowtech/wave/pull/138